swagger.yaml

```diff
@@ -1950,6 +1950,25 @@ definitions:
       discovery_agent_version:
         type: string
 
+  host_registration_response:
+    allOf:
+        - $ref: '#/definitions/host'
```

Comment:
Need to make sure that this is backwards compatible

Reply:
It is. All the environments will continue to work.